libraries/ml/src/string-measure.ts

```diff
@@ -2,12 +2,36 @@ import * as fl from 'fastest-levenshtein';
 import {jaroWinkler} from 'jaro-winkler-typescript';
 
 import {DistanceMetric} from '@datagrok-libraries/utils/src/type-declarations';
+import * as metric from '@datagrok-libraries/utils/src/similarity-metrics';
```

Comment:
It should not be included here. There should be more general class including both string and chemical measures in this ML library not to mix different entities.